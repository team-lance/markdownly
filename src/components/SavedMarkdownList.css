.list {
  position: absolute;
  top: 18em;
  list-style: none;
  font-family: 'Roboto Mono', monospace;
  color: #808080;
}

li {
  margin: 1em;
  font-size: 1.2em;
  text-transform: uppercase;
}

li:hover {
  color: black;
  font-weight: bold;
  cursor: pointer;
  text-decoration: underline;
}