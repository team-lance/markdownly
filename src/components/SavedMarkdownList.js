import React from 'react';
import PropTypes from 'prop-types';


export default function MarkdownTitles({ titles }) {

  const listOfTitles = titles.map((title, i)  => {
    return <li key={i}> {title}</li>;
  });

  return (
    <ul>
      {listOfTitles}
    </ul>
  );
}

MarkdownTitles.propTypes = {
  titles: PropTypes.array.isRequired
};
