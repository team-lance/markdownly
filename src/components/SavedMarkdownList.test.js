import React from 'react';
import { shallow } from 'enzyme';
import SavedMarkdownList from './SavedMarkdownList';

describe('SavedMarkdownList', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <SavedMarkdownList titles={['title1', 'title2']}/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
