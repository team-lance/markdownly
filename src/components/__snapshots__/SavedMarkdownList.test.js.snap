// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SavedMarkdownList matches a snapshot 1`] = `
<ul>
  <li
    key="0"
  >
     
    title1
  </li>
  <li
    key="1"
  >
     
    title2
  </li>
</ul>
`;
